NetCore/NetCoreRepro/ToDuplicate/Test.cs

```diff
@@ -1,24 +1,16 @@
 ﻿using System;
 using System.Diagnostics;
-using Moq;
 
 namespace NetCoreRepro.ToDuplicate
 {
     public class Test
     {
-        private readonly Mock<IClass> aClass;
-
-        public Test()
-        {
-            this.aClass = new Mock<IClass>();
-        }

```

Comment:
Ctor isn't needed at all. Newing up a `Moq.Mock<>` doesn't actually create a proxy object yet; that'll happen upon the first query to `mock.Object`, so doing all the work inside `DoSomething` should be equivalent to what we had before.